starlette/responses.py

```diff
@@ -112,7 +112,7 @@ def set_cookie(
         httponly: bool = False,
         samesite: typing.Optional[Literal["lax", "strict", "none"]] = "lax",
     ) -> None:
-        cookie: http.cookies.BaseCookie = http.cookies.SimpleCookie()
+        cookie: "http.cookies.BaseCookie[str]" = http.cookies.SimpleCookie()
```

Comment:
Why is this correct?

Reply:
Are you asking why `str` is the correct type? Both `key` and `value` are `str` (from the function signature) and `str` made type checker happy so pretty sure `str` is correct.

Reply:
As pointed on our conversation outside GitHub... It's because of the definition on typeshed: https://github.com/python/typeshed/blob/544c7c963333a4a415b951f13a6a6bbec2421161/stdlib/http/cookies.pyi#L54

Thanks @adriangb 😗